docs/processing_data.rst

```diff
@@ -152,6 +173,235 @@ Although this is a simple example of labeling data other things can be done here
+
+     +------------------+
+     | |flow_southmenu| |
+     +------------------+
+
+     - Readings - display the readings that are buffered in the storage service as a result of this pipeline.
+
+     - Export Readings - export the buffered readings for this pipeline to a CSV file.
+
+     - Delete - delete the service.
+
+  - **Display Readings** - display the readings that are buffered in the storage service as a result of this pipeline.
+
+  - **Show Logs** - display the logs written by this service.
+
+  - **Help** - show the online documentation of this south plugin. The online documentation will be shown in a new browser tab.
+
+These same controls and status indicators are also available in the south page that shows all of the current south services that are configured within the Fledge instance.
+
++--------------+
+| |flow_south| |
++--------------+
+
+Adding Filters
+~~~~~~~~~~~~~~
+
+Adding a new filter to a pipeline is a simple process of dragging the blank filter icon from the bottom left of the canvas and dropping it on one of the connecting lines in the filter pipeline.
+
++------------------+
+| |flow_addfilter| |
++------------------+
+
+Once the new filter has been dropped onto the connection, the pipeline will redraw and show an empty filter in the pipeline.
+
++--------------------+
+| |flow_filteradded| |
++--------------------+
+
+The dashed outline of the filter signifies that the filter has yet to be defined, its place within the pipeline is set, but the actual filter to be inserted as not been selected and it has not been configured. Clicking on the + icon will bring up the filter selection dialog.
+
++---------------------+
+| |flow_definefilter| |
++---------------------+
+
+Select the filter plugin to use from the list of plugins given. If a filter you need has not been installed you may install it by clicking on the *Install from available plugins* link below the list.
+
+Give the filter a name, names must be unique within the Fledge instance. Once complete click on the *Next* button to configure your chosen filter plugin.
+
+Here we have assumed you selected the *RMS* plugin and see the configuration screen for that plugin.
+
++---------------------+
+| |flow_filterconfig| | 
++---------------------+
+
+Configure your plugin and then click on *Done*. The filter screen will be displayed with the empty filter now replaced with your newly defined and configured filter.
+
++-------------------+
+| |flow_filterdone| |
++-------------------+
+
+.. note::
+
+   The colour of the filter components outline is used to distinguish the type of element it is. South plugins are shown in green and filters in yellow. Filters also display a "funnel" icon in the centre of the element.
+
+Similar controls are shown around the periphery of the filter icon to those shown on the south plugin. There is no count display when you hover over the filter and no readings or logs are available that related just to this filter.
```

Comment:
available that related just to this filter -> available that are related just to this filter